SP_README.md

```diff
@@ -0,0 +1,92 @@
+# Socialpoint LLVM Fork
+
+This repository contains a llvm fork. It contains some additional clang-tidy checks inside the socialpoint group of checks that may be useful to us. It also contains a modified ``run-clang-tidy.py`` that allows to exclude file paths from the checks done using a ``compile_commands.json``
+
+Currently we are interested only in 2 binaries generated from this repo: ``clang-tidy`` and ``clang-apply-replacements``
+
+## How can I run the clang-tidy checks?
+
+You need to build ``clang-tidy`` and ``clang-apply-replacements``. Don't mix the use of those binaries with others present in other distributions (like Xcode, Android NDK, LLVM standalone installation) because they may not work properly together
```

Comment:
Do we plan to build this on our build servers and make precompiled binaries available? 🤔

Reply:
I think that's the idea, but in the meanwhile I don't think it is bad to have this info here... There are some details that are a mess to put all together after a while without touching this